bindings/c/include/opendal.h

```diff
@@ -17,25 +17,33 @@
  * under the License.
  */
 
+#ifndef __OPENDAL_H__
+#define __OPENDAL_H__
 
-#ifndef _OPENDAL_H
-#define _OPENDAL_H
+/* Do not modify. This code was autogenerated by bindgen. */
 
-#include <stdint.h>
-#include <stddef.h>
+#include <stdarg.h>
 #include <stdbool.h>
+#include <stddef.h>
+#include <stdint.h>
+#include <stdlib.h>
 
-#ifdef __cplusplus
-extern "C" {
-#endif // __cplusplus
+typedef struct BlockingOperator BlockingOperator;
```

Comment:
I have no knowledge about async/blocking in the C ecosystem. How do they handle naming conventions?